src/types/redis_bloom_chain.cc

```diff
@@ -144,21 +150,19 @@ rocksdb::Status BloomChain::Add(const Slice &user_key, const Slice &item, int *r
   WriteBatchLogData log_data(kRedisBloomFilter, {"insert"});
   batch->PutLogData(log_data.Encode());
 
-  std::string item_string = item.ToString();
-
   // check
-  bool exist = false;
+  std::vector<bool> exist(1, false);                   // TODO: to refine in BF.MADD
   for (int i = metadata.n_filters - 1; i >= 0; --i) {  // TODO: to test which direction for searching is better
-    s = bloomCheck(bf_key_list[i], item_string, &exist);
+    s = bloomCheck(bf_key_list[i], {item}, &exist);
```

Comment:
It seems likely inconsistent return type between bloomCheck and MExists, the first one is vector<bool> and the later is vector<int>

Reply:
Sure, I think I will support `MAdd` in next patch, the return type of this funtion will be vector. And the `Add` functions will call `MAdd`

Reply:
OK, makes sense.